### DISCUSSION TITLE
Conversational Retrieval QA Chain - Input is not readable for chatbot

### DISCUSSION THREAD

**Discussion by flype223** [FlowiseAI/Flowise#1804] [Q&A]
Hi,

so, I followed this tutorial on Youtube to set everything up: https://www.youtube.com/watch?v=riXpu1tHzl0&t=303s
Everything is set up, like Api Keys, Index etc.. I also uploaded a PDF and hit the save button. 
But the chatbot does not have access to the PDF. I ask direct questions about the document and the response is Hmm, I am not sure or some general informations.

Someone has an assumption about the problem?

**Comment by guululu** [comment]
hi, did you have solutions to this issue? I'm also having the same problem

**Comment by alexandregcloficial** [comment]
I'm also having the same problem. Does anyone know more about this issue of the agent not returning specific questions from the attached document in Vector Store Retriever.
